### PR TITLE
chore: update discord instruction page content

### PR DESCRIPTION
**Pull request by mattwang44**
## Types of changes

* **Other (update page content)**

## Description

Update page based on [modified content](https://hackmd.io/@pycontw/ByoUdNir_) provided by program team.

### CODE REVIEW

File: pages/ext/discord.vue [hunk 1]
```diff
@@ -221,9 +273,97 @@ export default {
...
+                },
+                'r2-track': {
+                    name: '📖｜r2-track',
+                    roles: ['2021-staff', '2021-speaker', '2021-attendee'],
+                },
+                'r3-track': {
+                    name: '📖｜r3-track',
+                    roles: ['2021-staff', '2021-speaker', '2021-attendee'],
+                },
+                cathay: { name: '🦸｜國泰世華 Cathay United Bank' },
```
Comment (allen91wu) [inline pages/ext/discord.vue:355]:
```suggestion
                cathay: { name: '🦸｜國泰金控' },
```

File: pages/ext/discord.vue [hunk 2]
```diff
@@ -221,9 +273,97 @@ export default {
...
+                kloudless: { name: '🦸｜kloudless' },
+                esun: { name: '🦸｜玉山銀行 E.SUN Commercial Bank' },
+                'berry-ai': { name: '🦸｜berry-ai' },
+                sinopac: { name: '🦸｜永豐銀行 Bank SinoPac (China) Ltd.' },
+                appier: { name: '🦸｜appier' },
+                ichef: { name: '🦸｜ichef' },
+                'kkbox-group': { name: '🦸｜kkbox-group' },
+                onedegree: { name: '🦸｜onedegree' },
+                ctbc: { name: '🦸｜中國信託 CTBC Bank' },
```
Comment (allen91wu) [inline pages/ext/discord.vue:364]:
```suggestion
                ctbc: { name: '🦸｜中國信託' },
```

File: pages/ext/discord.vue [hunk 3]
```diff
@@ -221,9 +273,97 @@ export default {
...
+                tagtoo: { name: '🦸｜塔圖科技 Tagtoo' },
```
Comment (allen91wu) [inline pages/ext/discord.vue:365]:
```suggestion
                tagtoo: { name: '🦸｜塔圖科技' },
```

File: pages/ext/discord.vue [hunk 4]
```diff
@@ -221,9 +273,97 @@ export default {
...
+                    name: '📖｜r2-track',
+                    roles: ['2021-staff', '2021-speaker', '2021-attendee'],
+                },
+                'r3-track': {
+                    name: '📖｜r3-track',
+                    roles: ['2021-staff', '2021-speaker', '2021-attendee'],
+                },
+                cathay: { name: '🦸｜國泰世華 Cathay United Bank' },
+                kloudless: { name: '🦸｜kloudless' },
+                esun: { name: '🦸｜玉山銀行 E.SUN Commercial Bank' },
```
Comment (allen91wu) [inline pages/ext/discord.vue:357]:
```suggestion
                esun: { name: '🦸｜玉山銀行' },
```

File: pages/ext/discord.vue [hunk 5]
```diff
@@ -221,9 +273,97 @@ export default {
...
+                'berry-ai': { name: '🦸｜berry-ai' },
+                sinopac: { name: '🦸｜永豐銀行 Bank SinoPac (China) Ltd.' },
```
Comment (allen91wu) [inline pages/ext/discord.vue:359]:
```suggestion
                sinopac: { name: '🦸｜永豐金證券' },
```

**Review by allen91wu — COMMENTED** [review]
Hi @mattwang44,
Sorry for bothering you. I have left some comments about sponsor name typo.
Can you help me to modify these typos?

**Comment by mattwang44** [comment]
@allen91wu Thanks for your review! A new commit is added to solve this issue.

**Pull request merged**